src/Simulation/QCTraceSimulator/QCTraceSimulatorCore.cs

```diff
@@ -30,6 +30,13 @@ public class QCTraceSimulatorCoreConfiguration
         /// of the call graph.
         /// </summary>
         public uint CallStackDepthLimit = uint.MaxValue;
+
+        /// <summary>
+        /// Constrols if depth or width optimization is favored.
```

Comment:
Controls

Reply:
Updated. Here and in another configuration class.